doc/tor.md

```diff
@@ -211,3 +209,105 @@ for normal IPv4/IPv6 communication, use:
   Otherwise it is trivial to link them, which may reduce privacy. Onion
   services created automatically (as in section 2) always have only one port
   open.
+
+- In some regions of the world usage of the Tor network is monitored and/or blocked.
+  If you are in an environment that does not permit direct Tor connections or the use
+  of Tor bridges, then considering the trade-offs, it may not be safe to use Tor.
+
+- For maximum privacy, it is preferable to disable accepting incoming connections.
+
+- Users can download packages with [torsocks](https://gitlab.torproject.org/legacy/trac/-/wikis/doc/torsocks) for better security and privacy.
+
+- Trade-offs involved in using `onlynet=onion`:
+
+  The `onlynet=onion` configuration option can potentially ensure the node only attempts to connect over
+  the Tor network. The decision to use this option should consider the following trade-offs:
+
+  Only connecting over Tor is more private when you combine it with no reachable IPv4/IPv6 address,
+  in particular if you want to broadcast transactions without them being correlatable with your IP.
+
+  **Sybil Attacks**
+
+  On the other hand, if you only make random Tor connections, you're much more vulnerable to Sybil attacks.
+  As Tor addresses may be created at no cost, an attacker can potentially flood the network with many Tor
+  nodes and receive all of the outbound Tor connections an `onlynet=tor` node makes.
+
+  This is significantly less a concern with IPv4/IPv6 (especially with asmap) due to cost of getting IPs in many
```

Comment:
```suggestion
  This is significantly less a concern with IPv4/IPv6 (especially with asmap) due to the cost of obtaining IPs in many
```

Reply:
Done. Made changes in https://github.com/bitcoin/bitcoin/commit/6bf33db373588582c85ee537cd5e9302c593cb02